### ISSUE TITLE
BUG: Mumble Stopped Working (user error)

### ISSUE THREAD

**Issue by AtmanActive** [mumble-voip/mumble#6086]
### Description

So I got the brilliant idea to add two shortcuts for volume up and volume down so I can control Mumble's output volume while I'm in the game.
Made a mistake of not double-checking that those keys bound are not used elsewhere.
Few hours later, I unintentionally press volume down button and Mumble gets into state where I can't hear no one, but also, none of the incoming LEDs would blink.
It looked like completely bugged.
After two days of Mumble not working for me and getting all my peers mad, I finally rememberred this volume up/down addition by me, went to settings, removed the shortcut bindings, saw that Mumble's output volume is indeed set to zero, and fixed the situation by returning it to 100%.

My report here is about how Mumble's UI handled that situation in a most unfortunate way.

1.) Why would incoming LEDs stop working when Mumble's master volume is set to zero? That's completely unintuitive. There's no device or software in this world that behaves like that. That would be comparable to me lowering volume on my phone and then my phone turning off the screen when volume reaches zero?!?

2.) Since Mumble's master volume is not shown anywhere on the UI, but visible only in the settings, in my humble opinion, there should be some kind of an indicator or color change or something when it is set to zero, to make user aware that no sound will come out of it.

Just wanted to share my bad experience with this.

Thanks.

### Steps to reproduce

1.) set shortcuts for volume down and volume up
2.) get on a call
3.) press volume down a few times
4.) wonder why is Mumble behaving weird

### Mumble version

1.4.287

### Mumble component

Client

### OS

Windows

### Reproducible?

Yes

**Comment by Krzmbrzl** [comment]
I agree that this is an unfortunate way to handle this.

**Comment by kbanushi** [comment]
Hi,

I'd like to work on this and I think the two issues you brought up are pretty fair.  Something else I was thinking about was that this probably would have been avoided if there was a popup alerting the user when they add a new shortcut that uses the same key as a different shortcut.  If that doesn't already exist I would like to work on that too.

**Comment by Hartmnt** [comment]
> Something else I was thinking about was that this probably would have been avoided if there was a popup alerting the user when they add a new shortcut that uses the same key as a different shortcut. If that doesn't already exist I would like to work on that too.

I think the problem here is that the shortcut keys were used in an application outside of Mumble e.g. a game. I do not think there is a (practical) way to detect, if a global shortcut is overlapping with a key combination of another application.

The main point of this issue is the speech indicator which should **not** be looking like no one is talking, if the volume in Mumble is set to 0 client-side. There is already a speech indicator icon for "this client is locally muted" which theoretically could also be applied to all clients, if the volume is set to 0.

**Comment by AtmanActive** [comment]
How about a label suffix, similar to Local Volume Adjustment: when I want to hear more (or less) of some of the participants, then I right-click on them, choose Local Volume Adjustment and from then on, their handle in the list is suffixed with "[-6]" or whatever value I set. Similarly, when my own Mumble output volume is not at 100%, my own handle should be suffixed with "(out:30%)" or "(out:0%)" or something similar.
That would fix the confusion for sure.

**Comment by Hartmnt** [comment]
> That's the main source of confusion: instead of somehow showing ME as muted, Mumble shows EVERYONE ELSE as mute.

Indeed, but the option here is to either show the regular, green "this client is talking" icon or the "this client is talking while locally muted" icon for every client. Showing the "this client is not talking" icon is unacceptable. I also like the addition of your local "out" volume text next to the name.

**Comment by Krzmbrzl** [comment]
> Similarly, when my own Mumble output volume is not at 100%, my own handle should be suffixed with "(out:30%)" or "(out:0%)" or something similar.

I think that kind of thing would be best appended to the window's titlebar. Writing it into the UI next to the local user seems inappropriate as that would indicate that this property is related to the user-objects, whereas it's a global setting that is in effect.
